src/python_testing/TC_EWATERHTR_2_2.py

```diff
@@ -235,6 +241,16 @@ async def test_TC_EWATERHTR_2_2(self):
         self.step("8b")
         await self.check_whm_attribute("BoostState", Clusters.WaterHeaterManagement.Enums.BoostStateEnum.kActive)
 
+        ################################################################################
+        event_data = events_callback.wait_for_event_report(Clusters.WaterHeaterManagement.Events.BoostStarted)
```

Comment:
Need to update the TestStep above to match and then we update the test plan.

Reply:
Think this is done